spec/statement.dd

```diff
@@ -808,60 +814,180 @@ $(GNAME OpApplyParameter):
     $(P This would print:)
 
 $(CONSOLE
-73
-82
+1
+2
+3
+4
 )
-    $(PANEL
-    The `scope` storage class on the $(D dg) parameter means that the delegate does
-    not escape the scope of the $(D opApply) function (an example would be assigning $(D dg) to a
-    global variable). If it cannot be statically guaranteed that $(D dg) does not escape, a closure may
-    be allocated for it on the heap instead of the stack.
 
-    $(BEST_PRACTICE Annotate delegate parameters to `opApply` functions with `scope` when possible.)
+    $(BEST_PRACTICE Annotate delegate parameters to apply functions with `scope` when possible.)
```

Comment:
![grafik](https://github.com/dlang/dlang.org/assets/10766348/dd019e9e-0d95-44ed-ba1d-c957e2b7bb8f)
I know exactly why I changed it. The “9.” looks bad. I honestly have no issue with it going after the PANEL, it would look like that (faked with HTML DOM manipulation):
![grafik](https://github.com/dlang/dlang.org/assets/10766348/1c3b64b3-18ee-4548-bea5-c19641a70616)